modules/ROOT/pages/tuning-domains.adoc

```diff
@@ -0,0 +1,17 @@
+= Domains
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Besides the listed benefits on the Shared Resources MuleSoft documentation, domains are also a performance choice when deploying multiple services on the same on-premises Mule runtime engine (Mule). By having a centralized point for all the shared resources, domains help the class-loading process, therefore metaspace memory usage, to be more efficient, especially because dependencies of the domain (those declared in the `pom.xml` file) are also shared in the domain apps.
+
+== Example using Domain
+To give an example, consider having a set of HTTP Proxy applications that also apply one API Gateway policy, such as IP whitelist. Each application interacts with the same backend.
+
+In a Mule configured to have 2GB of heap memory and 256MB of metaspace, deploying those applications independently without using a proper domain, starts having noticeable TPS (transactions per second) drops when deploying 60+ of these applications due to saturated resources, and around 80 applications start to have deployment issues due to out of memory exceptions. +
```

Comment:
Just in case you feel you must use three paragraphs...:
```suggestion
In a Mule instance with 2 GB of heap memory and 256 MB of metaspace, deploying over 60 of such applications independently shows a noticeable reduction in TPS (transactions per second) due to saturated resources. At 80 applications, you see deployment issues due to out-of-memory exceptions. +
```